make.js

```diff
@@ -144,8 +144,13 @@ target.test = function() {
 
     var suiteArg = process.argv[4];
     var suiteType = suiteArg || 'L0';  
-    var testsSpec = path.join(buildPath, "/**/Tests", suiteType + ".js");
-    run("mocha " + testsSpec, true);
+
+    var taskArg = process.argv[5];
+    var taskType = taskArg || '**';
+
+    // the tests may be under _build or _common
+    var testsSpec = path.join(__dirname, '_[bc]*', taskType, 'Tests', suiteType + ".js");
+    run(`mocha ${testsSpec}`, true);
```

Comment:
Another alternative is not to separate have a separate _common directory, but dump the common modules into the _build folder as well. It would mean that during the packaging step, they would need to be excluded somehow (e.g. skip if they don't have a task.json or aren't in the "taskList").